pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -785,3 +803,24 @@ func (c *Cluster) getOSDLabels(osdID int, failureDomainValue string, portable bo
 		portableKey:         strconv.FormatBool(portable),
 	}
 }
+
+//GetVolumeGroupName returns the Volume group name from the given Logical Volume Path
```

Comment:
What do you expect to receive as `osd.LvPath`? There are many symlinks under `/dev` that would point to the device. For example, on my laptop I have a VG calles `srv` and it has a LV `libvirt`:
```
[ndevos@ndevos-x270 ~]$ ls -l /dev/srv/libvirt 
lrwxrwxrwx. 1 root root 7 Aug 28 17:31 /dev/srv/libvirt -> ../dm-5
[ndevos@ndevos-x270 ~]$ ls -l /dev/mapper/srv-libvirt 
lrwxrwxrwx. 1 root root 7 Aug 28 17:31 /dev/mapper/srv-libvirt -> ../dm-5
[ndevos@ndevos-x270 ~]$ ls -l /dev/dm-5
brw-rw----. 1 root disk 253, 5 Aug 28 17:31 /dev/dm-5
```
So, valid device nodes to use the LV are:

* /dev/dm-5
* /dev/srv/libvirt
* /dev/mapper/srv-libvirt

Some verification on what device node is passed is recommended.

Reply:
We are obtaining the lvpath from the output of `ceph lvm prepare` command ([here](https://github.com/rook/rook/blob/7eef87fbcba422f7f1009203bc1dd652c25ea1df/pkg/daemon/ceph/osd/volume.go#L121)).  So its in the format `/dev/<volume group name>/<logical volume name>`

Reply:
Right, that looks good. A comment explaining that (or at least the format) would be nice.